app/src/main/java/org/oppia/android/app/home/HomeViewModel.kt

```diff
@@ -27,8 +31,10 @@ import org.oppia.android.util.parser.StoryHtmlParserEntityType
 import org.oppia.android.util.parser.TopicHtmlParserEntityType
 import org.oppia.android.util.system.OppiaClock
 
-private const val PROFILE_AND_ONGOING_STORY_COMBINED_PROVIDER_ID = "profile+ongoingStoryList"
-private const val HOME_FRAGMENT_COMBINED_PROVIDER_ID = "profile+ongoingStoryList+topicListProvider"
+private const val PROFILE_AND_Recommended_Activity_COMBINED_PROVIDER_ID =
```

Comment:
Should this be "PROFILE_AND_RECOMMENDED_ACTIVITY_COMBINED_PROVIDER_ID"?

Also, this might be confused with recommended stories -- since this could be any of recommended stories, recently played stories, or last-played stories I think using "Promoted Activity" instead of "Recommended Activity" might help with clarity on this

Reply:
This sounds good to me. As it is confusing. Recommended activity has 2 types i.e `Coming Soon topics` and `Recommended stories`. Recommended stories intern contains 3 types `Recommended, Recently-played and Last-played stories`. @Rajat and @BenHenning please clarify on this.

Reply:
@jcqli ' suggestion is correct. Nice catch.

Reply:
FYI @veena14cs Please edit your comment as you have added some other developer instead of me.

Reply:
@rt4914 edited my comment. 
So I shall I rename it everywhere else, wherever we are using Recommended Activity to Promoted Activity.

Reply:
Done.